platform.json

```diff
@@ -192,7 +192,7 @@
       "type": "framework",
       "optional": true,
       "owner": "platformio",
-      "version": "~4.20000.0"
+      "version": "~4.30000.0"
```

Comment:
Please delete this line as I won't be able to merge this PR when the Arduino core v2.1.0 is ready.

Reply:
Sorry, got distracted by something. Fixing now.